test/aiecc/simple.mlir

```diff
@@ -32,12 +32,14 @@
 // NOCOMPILE-NOT: {{^[^ ]*llc}}
 
 module {
-  %12 = aie.tile(1, 2)
-  %buf = aie.buffer(%12) : memref<256xi32>
-  %4 = aie.core(%12)  {
-    %0 = arith.constant 0 : i32
-    %1 = arith.constant 0 : index
-    memref.store %0, %buf[%1] : memref<256xi32>
-    aie.end
+  aie.device(xcvc1902) {
```

Comment:
Is this not added by the device canonicalization pass?

Reply:
IIRC it is. However this is something I wanted to revisit at some point. Maybe vck190 isn't the best default. And perhaps devices could be inserted/modified by flags to aiecc.py at compile time rather than at MLIR generation from Python or design time.